design/api.go

```diff
@@ -0,0 +1,70 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.API("feature", func() {
+	a.Title("Feature Toggles Service")
+	a.Description("List feature toggles service")
+	a.Version("1.0")
+	a.Host("openshift.io")
+	a.Scheme("http")
+	a.BasePath("/")
+	a.Consumes("application/json")
+	a.Produces("application/json")
+
+	a.License(func() {
+		a.Name("Apache License Version 2.0")
+		a.URL("http://www.apache.org/licenses/LICENSE-2.0")
+	})
+	a.Origin("/[.*openshift.io|localhost]/", func() {
+		a.Methods("GET", "POST", "PUT", "PATCH", "DELETE")
+		a.Headers("X-Request-Id", "Content-Type", "Authorization")
+		a.MaxAge(600)
+		a.Credentials()
+	})
+
+	a.Trait("GenericLinksTrait", func() {
+		a.Attribute("self", d.String)
+		a.Attribute("related", d.String)
+		a.Attribute("meta", a.HashOf(d.String, d.Any))
+	})
+
+	a.Trait("jsonapi-media-type", func() {
+		a.ContentType("application/vnd.api+json")
+	})
+
+	a.Trait("conditional", func() {
+		a.Headers(func() {
+			a.Header("If-Modified-Since", d.String)
+			a.Header("If-None-Match", d.String)
+		})
+	})
+
+	//a.JWTSecurity("jwt", func() {
```

Comment:
we'll need `jwt` to handle the user's token in the requests, so this needs to be commented out